requirements.txt

```diff
@@ -10,11 +11,12 @@ boto3==1.9.199
 owslib~=0.18.0
 requests_file~=1.4.3
 natsort~=6.0.0
-pillow~=6.1.0
-pyproj==1.9.6
-psycopg2
+pillow
+pyproj
+psycopg2 ; sys_platform == 'linux'
+psycopg2-binary ; sys_platform == 'darwin'
```

Comment:
This accommodates macOS users without the need to explain a manual step in the README.